Classes/KIFUITestActor.m

```diff
@@ -1164,7 +1168,7 @@ - (void)pullToRefreshAccessibilityElement:(UIAccessibilityElement *)element inVi
     // Can handle only the touchable space.
     CGRect elementFrame = [viewToSwipe convertRect:viewToSwipe.bounds toView:[UIApplication sharedApplication].keyWindow.rootViewController.view];
     CGPoint swipeStart = CGPointCenteredInRect(elementFrame);
-    CGPoint swipeDisplacement = CGPointMake(CGRectGetMidX(elementFrame), CGRectGetMaxY(elementFrame));
+    CGPoint swipeDisplacement = CGPointMake(CGRectGetMidX(elementFrame) - swipeStart.x, CGRectGetMaxY(elementFrame) - swipeStart.y);
```

Comment:
What change led to this bit? Why has the displacement changed?

Reply:
Oops didn't mean to change this.

The reason why i even put this in here at all was when I put in the visualizer i noticed the the pull to refresh went diagonally this fixes that. Not an issue so going to leave as is for now but just was testing things out and forgot to remove it